feature/space-entrepreneurship-funding-sources.md

```diff
@@ -5,11 +5,10 @@ menu: true
 order: 4
 ---
 
-We were looking for a directory of funding resources for NewSpace startups and couldn't find one. So we decided to start our own.
+*We were looking for a directory of funding resources for NewSpace startups and couldn't find one. So we decided to start our own.*
 
-This open document has everything we have found so far, but it is definitely not complete. [Please email us about resources we've missed](mailto:groundcontrol@orbitalindex.com) (or [submit a Pull Request on GitHub](https://github.com/orbitalindex/orbital-index))! ❤️
+This open document has everything we've found so far, but it is definitely not complete. [Please email us about resources we've missed](mailto:groundcontrol@orbitalindex.com) (or [submit a Pull Request on GitHub](https://github.com/orbitalindex/orbital-index))! ❤️
```

Comment:
Should we note here that by contributing you're waiving any copyright to submitted content?

Reply:
We don't do it on Awesome Space—perhaps we should, but I think this kind of very short contribution is pretty safe to assume people know it's public domain.